src/app.ts

```diff
@@ -41,6 +40,9 @@ if (process.env.NODE_ENV === 'dev') {
     autoSwagger(RoutesWrapper.declareRoutes());
 }
 
+// eslint-disable-next-line import/first
+import SwaggerDocument from '../api-docs/swagger-doc.json';
```

Comment:
Seria interessante mover o import para o início, assim evita-se desativar regras in-line

Reply:
Resolvido